### PR TITLE
Update integration tests following event rename

### PR DESCRIPTION
**Pull request by gilescope**
Update tests: ump.ExecutedUpward now called messageQueue.Processed

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2953603

File: parachains/integration-tests/e2e/assets/asset-hub-kusama/0_xcm/2_ump.yml [hunk 1]
```diff
@@ -97,10 +97,17 @@ tests:
...
                       threshold: *weight_threshold
-                      result: [{ Complete: { refTime: '298,716,000', proofSize: 0 }}]
+                      result:
+                        origin:
+                        Ump:
+                          Para: '1,000'
+                        weightUsed:
+                          refTime: '298,716,000'
+                          proofSize: '0'
+                        success: true
```
Comment (NachoPal) [inline parachains/integration-tests/e2e/assets/asset-hub-kusama/0_xcm/2_ump.yml:110]: Do you mind to use the string format for `results` as it is done for the rest of tests to be consistent?

**Review by ggwpez — APPROVED** [review]
Nice, I was waiting for this to trickle down :laughing:

**Review by muharem — APPROVED** [review]

**Comment by gilescope** [comment]
bot merge

File: parachains/integration-tests/e2e/assets/asset-hub-kusama/0_xcm/2_ump.yml (hunk 1 above)
Comment (gilescope) [reply parachains/integration-tests/e2e/assets/asset-hub-kusama/0_xcm/2_ump.yml:110]: will be in next PR.

**Pull request merged**